src/js/ads.js

```diff
@@ -32,12 +32,12 @@ class VideoAds {
 				document.getElementsByTagName("head")[0].appendChild(googleSdkScript);
 			}
 
-			if(googleSdkScript.hasAttribute('data-o-video-script-loaded')) {
+			if (this.sdkScriptLoaded) {
 				resolve();
 			} else {
 
 				googleSdkScript.addEventListener('load', () => {
-					googleSdkScript.setAttribute('data-o-video-script-loaded', true);
+					this.sdkScriptLoaded = true;
```

Comment:
That would be different for each instance of Ads, which is different for each video. I mean something like VideoAds.sdkScriptLoaded. I think that would work